File: tendayFoundation.D/DAY8/prime_check.c
#include <stdio.h>

int main(){
    int n = 7, flag = 0;
    if (n <= 1) {
        flag = 1;
    } else {
        for(int i = 2; i <= n / 2; i++){
            if(n % i == 0){
                flag = 1;
                break;
            }
        }
    }

    if(flag == 0){
        printf("%d is a prime number\n", n);
    } else {
        printf("%d is not a prime number\n", n);
    }
    return 0;
}

File: JAVA/prime_check.java
import java.util.Scanner;

public class prime_check {
    public static void main(String a[]) {
        Scanner s = new Scanner(System.in);
        System.out.println("Enter the number to check: ");
        int isPrime = 0;
        int num = s.nextInt();
        for (int i = 2; i <= num / 2; i++) {
            if (num % i == 0) {
                isPrime = 1;
                break;
            }

        }
        if (isPrime == 0) {
            System.out.println("Entered number is prime");
        } else {
            System.out.println("Entered number is not prime");
        }
        s.close();

    }
}
